src/main/webapp/view/transcript/transcript.js

```diff
@@ -59,6 +61,8 @@ export class TranscriptLineElement extends HTMLElement {
         ['justify-content-start', 'mb-1', 'transcript-line-timestamp']);
     TranscriptLineElement.appendParagraphToContainer(
         transcriptLine.content, contentDivElement, ['ml-4', 'mb-1']);
+    contentDivElement.appendChild(
```

Comment:
I assume you'll need to save a reference to this for incrementing later?

Reply:
I was planning on querying for it, but saving a reference would be faster. Thanks! updated.